Formula/sbt.rb

```diff
@@ -20,11 +20,30 @@ def install
       s.gsub! "/etc/sbt/sbtopts", "#{etc}/sbtopts"
     end
 
-    inreplace "bin/sbt-launch-lib.bash", "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+    inreplace "bin/sbt-launch-lib.bash" do |s|
+      s.gsub! "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+      if s.include?("${sbt_bin_dir}/java9-rt-export.jar")
+        s.gsub! "${sbt_bin_dir}/java9-rt-export.jar", "#{libexec}/java9-rt-export.jar"
```

Comment:
I'm following the existing pattern of calling `inreplace` to change where `sbt_home` (or `sbt_bin_dir`) appear in the original bash script. These are calculated in the original as follows:

```
declare -r sbt_bin_dir="$(dirname "$(realpath "$0")")"
declare -r sbt_home="$(dirname "$sbt_bin_dir")"
```

I'm guessing these are needed because the bash files and jar files exist directly under `libexec` while normal `sbt` directory would have `bin`, `config`, and `lib` directory under it.